### DISCUSSION TITLE
i got error "request failed with status code 500: malformed utf-8 characters, possibly incorrectly encoded"

### DISCUSSION THREAD

**Discussion by nguyenhung220598** [opcodesio/log-viewer#288] [Q&A]
When I log with sql code local.ERROR: SQLSTATE[22007] with data where the value is Japanese text, it seems that it is not encoded in UTF-8 format, is there any way to fix it?

**Comment by arukompas** [comment]
Hey @nguyenhung220598 please try upgrading to the latest version, v3.1.8, which should contain a fix :) let me know if it helps!
